lib/screens/explore_screens/unplant_seeds/unplant_seeds_screen.dart

```diff
@@ -76,9 +109,8 @@ class UnplantSeedsScreen extends StatelessWidget {
                         alignment: Alignment.bottomCenter,
                         child: FlatButtonLong(
                           title: 'Unplant Seeds',
-                          enabled: false,
-                          //Next pr
-                          onPressed: () => {},
+                          enabled: state.isUnplantSeedsButtonEnabled,
+                          onPressed: () => {BlocProvider.of<UnplantSeedsBloc>(context).add(OnUnplantSeedsButtonTap())},
```

Comment:
Amm why these { }

Reply:
change 
` () => {BlocProvider.of<UnplantSeedsBloc>(context).add(OnUnplantSeedsButtonTap())},`
 to 
`  () => BlocProvider.of<UnplantSeedsBloc>(context).add(OnUnplantSeedsButtonTap()),`